__tests__/app.test.js

```diff
@@ -70,3 +70,57 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:articleid", () => {
+  test("responds with status code 200", () => {
+    return request(app).get("/api/articles/6").expect(200);
+  });
+  test("responds with an article object", () => {
+    return request(app)
+      .get("/api/articles/5")
+      .then(({ body }) => {
+        expect(typeof body).toBe("object");
+      });
+  });
+  test("responds with an article object with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .then(({ body }) => {
+        expect(body.article.hasOwnProperty("title")).toBe(true);
+        expect(body.article.hasOwnProperty("article_id")).toBe(true);
+        expect(body.article.hasOwnProperty("body")).toBe(true);
+        expect(body.article.hasOwnProperty("topic")).toBe(true);
+        expect(body.article.hasOwnProperty("created_at")).toBe(true);
+        expect(body.article.hasOwnProperty("votes")).toBe(true);
+        expect(body.article.hasOwnProperty("article_img_url")).toBe(true);
+      });
+  });
```

Comment:
This test is redundant, to make it pass you still need to basically send the right data back on the response so you may as well go straight into testing the response object is correct like below

Reply:
removed this test